apps/prairielearn/src/pages/home/home.sql

```diff
@@ -1,11 +1,12 @@
--- BLOCK select_home
+-- BLOCK select_instructor_courses
 WITH
   example_courses AS (
     SELECT
       c.short_name || ': ' || c.title AS label,
```

Comment:
You might consider doing the formatting of this (both here and in the student courses query) in the page template instead of in the DB. This is a minor thing and I definitely won't block on it.

Reply:
Done in b759a7b7a.